web/src/ProviderListPage.js

```diff
@@ -69,11 +71,45 @@ class ProviderListPage extends BaseListPage {
     };
   }
 
+  newChat() {
+    const randomName = Setting.getRandomName();
+    return {
+      owner: "admin",
+      name: `chat_${randomName}`,
+      createdTime: moment().format(),
+      updatedTime: moment().format(),
+      organization: this.props.account.owner,
+      displayName: `${i18next.t("chat:New Chat")} - ${randomName}`,
+      category: i18next.t("chat:Default Category"),
+      type: "provider",
+      user: this.props.account.name,
+      user1: "",
+      user2: "",
+      users: [],
+      clientIp: "",
+      userAgent: "",
+      messageCount: 0,
+      tokenCount: 0,
+      needTitle: true,
+      isHidden: true,
+    };
+  }
+
   addProvider(needStorage = false) {
-    let newProvider = this.newProvider();
+    const newChat = this.newChat();
```

Comment:
Not here, not every provider needs a chat

We don't need to modify ProviderListPage.js